src/common/Smi_Common_Python/SmiServices/DicomText.py

```diff
@@ -84,14 +37,22 @@ class DicomText:
     _include_unexpected_tags = False # SemEHR does not use unknown tags anyway so ignore them
     _warn_unexpected_tag = False     # print if an unexpected tag is encountered
     _replace_HTML_entities = True    # replace HTML tags with same length of space chars
```

Comment:
Is this actually about entities (such as &amp;pound;) or tags? The only entity I saw handled was &nbsp;, the rest seem to be ignored?